packages/@aws-cdk/aws-appmesh/lib/gateway-route.ts

```diff
@@ -151,18 +148,26 @@ class ImportedGatewayRoute extends cdk.Resource implements IGatewayRoute {
   public gatewayRouteArn: string;
 
   /**
-   * The VirtualGateway the GatewayRoute belongs to
+   * The VirtualGateway the GatewayRoute is associated with
    */
   public virtualGateway: IVirtualGateway;
 
-  constructor(scope: Construct, id: string, props: GatewayRouteAttributes) {
+  constructor(scope: Construct, id: string, attrs?: GatewayRouteAttributes, gatewayRouteArn?: string) {
```

Comment:
change to props?

Reply:
I can do that. I changed it since the `fromAttributes` methods asked for the parameter to be named `attrs`

Reply:
Is there a general rule of thumb when it should be `props` vs `attrs`?

Reply:
I think what Elad is saying is to add an `ImportedGatewayRouteProps` interface that extends `GatewayRouteAttributes` and contains the `gatewayRouteArn` property, instead of having 4 arguments to this constructor.

Reply:
Got it. Makes sense